backend/src/server/routes/v1/password-router.ts

```diff
@@ -80,7 +80,9 @@ export const registerPasswordRouter = async (server: FastifyZodProvider) => {
     method: "POST",
     url: "/email/password-reset",
     config: {
-      rateLimit: authRateLimit
+      rateLimit: smtpRateLimit({
+        keyGenerator: (req) => (req.body as { email: string }).email
```

Comment:
maybe we want to trim so that anything path isn't taken? otherwise redis memory might fill up